spec/shop_spec.rb

```diff
@@ -4,14 +4,22 @@
 describe Shop do
   it 'responds to name' do #description of your spec/test
     shop = Shop.new
-
     expect(shop).to respond_to(:name)
     #expects shop instance to have an name accessor/method
   end
 
-  it 'sums two numbers' do
-    result = Shop.sum(3,4)
+describe "#create" do
+  it 'increments data row by 1' do
+    shop = Shop.new
+    expect{shop.create}.to change{shop.count}
+  end
+end
 
-    expect(result).to eq(7)
+describe "#name_matches" do
+  it 'should be true if it matches' do
```

Comment:
related to the specs descriptions, `should` is not used anymore, currently, we write more like: it matches, it returns, it does, etc.